nixos/modules/programs/steam.nix

```diff
@@ -29,6 +30,23 @@ in {
       '';
     };
 
+    env = mkOption {
+      type = types.attrs;
+      description = mdDoc ''
+        Environment variables to run Steam with.
+
+        Uses `lib.strings.toShellVars` internally; see the documentation for more info.
```

Comment:
Can we move this to apply and defaultText to make it obvious what happens?

Reply:
I'm not sure what you mean by that.